backend/storj/fs.go

```diff
@@ -22,32 +22,80 @@ import (
 	"storj.io/uplink"
 )
 
+var satMap = map[string]string{
+	"us-central-1.tardigrade.io":  "12EayRS2V1kEsWESU9QMRseFhdxYxKicsiFmxrsLZHeLUtdps3S@us-central-1.tardigrade.io:7777",
+	"europe-west-1.tardigrade.io": "12L9ZFwhzVpuEKMUNUqkaTLGzwY9G24tbiigLiXpmZWKwmcNDDs@europe-west-1.tardigrade.io:7777",
+	"asia-east-1.tardigrade.io":   "121RTSDpyNZVcEU84Ticf2L1ntiuUimbWgfATz21tuvgk3vzoA6@asia-east-1.tardigrade.io:7777",
+}
+
 // Register with Fs
 func init() {
 	fs.Register(&fs.RegInfo{
 		Name:        "storj",
 		Description: "Storj Decentralized Cloud Storage",
 		NewFs:       NewFs,
+		Config: func(name string, configMapper configmap.Mapper) {
+			satelliteString, _ := configMapper.Get("satellite-address")
+			apiKey, _ := configMapper.Get("api-key")
+			passphrase, _ := configMapper.Get("passphrase")
+			accessString, _ := configMapper.Get("access")
+
+			config.FileDeleteKey(name, "satellite-address")
+			config.FileDeleteKey(name, "api-key")
+			config.FileDeleteKey(name, "passphrase")
+
+			if accessString != "" {
+				return
+			}
+
+			satellite, found := satMap[satelliteString]
+			if !found {
+				satellite = satelliteString
+			}
+
+			access, err := uplink.RequestAccessWithPassphrase(context.TODO(), satellite, apiKey, passphrase)
+			if err != nil {
+				fs.Errorf(nil, "Couldn't create access grant: %v", err)
+				return
+			}
+
+			serialziedAccess, err := access.Serialize()
+			if err != nil {
+				fs.Errorf(nil, "Couldn't serialize access grant: %v", err)
+				return
+			}
+
+			configMapper.Set("access", serialziedAccess)
+		},
 		Options: []fs.Option{
 			{
 				Name:     "access",
-				Help:     "Uplink access.",
+				Help:     "Access grant. Leave it blank if you want to use your API Key",
 				Required: false,
```

Comment:
Do we also want to set the is password flag here?

Reply:
if we do this we should do the same thing for API Key, my main concern is that it will look strange because for Access, API Key and Passphrase we will be asked to repeat input, might be bad UX

Reply:
yeah, I have the same reservations